### ISSUE TITLE
Improvements to bitbuffer_extract_bytes()?

### ISSUE THREAD

**Issue by roger-** [merbanan/rtl_433#847]
I noticed that `bitbuffer_extract_bytes()` will sometimes copy more bits than you requested; not sure if that's intentional. Also would be nice if supported different endianess options.

**Comment by zuckschwerdt** [comment]
Yes, bottom bits should be masked out. If you extract on a byte boundary that might not be the case. I'll take a look.
But endianess? Don't go there, rtl_433 doesn't care about that and you probably should work byte-wise in any case.
